var/spack/repos/builtin/packages/mvapich2/mpir_attr_delete_list_segfault.patch

```diff
@@ -0,0 +1,18 @@
+--- a/src/mpi/attr/attrutil.c
++++ b/src/mpi/attr/attrutil.c
+@@ -266,6 +266,7 @@
+ 	   corresponding keyval */
+ 	/* Still to do: capture any error returns but continue to 
+ 	   process attributes */
++    if (p->keyval) {
+ 	mpi_errno = MPIR_Call_attr_delete( handle, p );
+ 
+ 	/* We must also remove the keyval reference.  If the keyval
+@@ -282,6 +283,7 @@
+ 		MPIU_Handle_obj_free( &MPID_Keyval_mem, p->keyval );
+ 	    }
+ 	}
++	}
```

Comment:
A super small OCD nitpick. `    if (p->keyval) {` starts with 4 spaces, but then the closing bracket ends with a tab. Would it be possible to update these to both either use tabs or spaces? This isn't Spack code so I'm happy to merge this would be complicated, but figured I'm mention it to satisfy my own brain.

Reply:
The source code of MVAPICH mixes up spaces and tabs 🙃

Reply:
Eww! Hahaha oh well